pages.hi/common/ln.md

```diff
@@ -0,0 +1,16 @@
+# ln
+
+> फाइलों और निर्देशिकाओं के लिंक बनाता है।
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>।
```

Comment:
```suggestion
> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>.
```

Reply:
'।' is actually the full stop alternative in hindi, are we sure that we want to use '.' here?

Reply:
We should keep what's correct. So it should be `!`.